### PR TITLE
wallet connect cp/paste flow

### PR DESCRIPTION
**Pull request by rinzler99**

### CODE REVIEW

File: src/utils/enums.js [hunk 1]
```diff
@@ -927,7 +927,8 @@ export const COINGECKO_SUPPORTED_CURRENCIES = new Set([
 
 export const WALLET_CONNECT_CARD_DATA = {
   [LOCALE_EN]: {
-    title: 'Explore & Connect to dapps via Wallet Connect',
+    title: 'Connect to Web3 Apps via WalletConnect',
+    caption: 'Paste the WalletConnect QR code on Web3 App below to connect',
```
Comment (chaitanyapotti) [inline src/utils/enums.js:931]: Add captions for other languages as well

Comment (chaitanyapotti) [reply src/utils/enums.js:931]: or move to airtable

**Comment by rinzler99** [comment]
workin demo : https://www.loom.com/share/d61ce7447d4249ca95e561f8a5bdbf1e

Figma File : https://www.figma.com/file/3p3IoglxHHlry87ojz8htF/Torus-Wallet?node-id=1201%3A11189

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**